CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Use default config name as service name and allow both service name and module name
```

Comment:
this doesn't belong in the changelog, and is also not breaking - it restores previous functionality